dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -3156,67 +3405,38 @@ def snapshot_page(n_clicks, url, wait_selector):
         For more information see <dccLink href="/dash-enterprise/application-structure" children="Application Structure"/>.
 
         &nbsp;
-        ''')
-    ]),
+        '''
+    ),
 
-    html.Details([
-        html.Summary("Failed to find application object 'server' in 'app"),
+    rc.Markdown(
+        '''
+        ### SSH deploy: git push is asking for password
+        '''
+    ),
 
-        rc.Markdown(
+    rc.Markdown(
         '''
         ```shell
-        ...
-        remote:        Failed to find application object 'server' in 'app'
-        remote:        [2018-08-16 16:00:49 +0000] [181] [INFO] Worker exiting (pid: 181)
-        remote:        [2018-08-16 16:00:49 +0000] [12] [INFO] Shutting down: Master
-        remote:        [2018-08-16 16:00:49 +0000] [12] [INFO] Reason: App failed to load.
-        remote:        [2018-08-16 16:00:51 +0000] [12] [INFO] Starting gunicorn 19.9.0
-        remote:        [2018-08-16 16:00:51 +0000] [12] [INFO] Listening at: http://0.0.0.0:5000 (12)
-        remote:        [2018-08-16 16:00:51 +0000] [12] [INFO] Using worker: sync
-        remote:        [2018-08-16 16:00:51 +0000] [179] [INFO] Booting worker with pid: 179
-        remote:        [2018-08-16 16:00:51 +0000] [180] [INFO] Booting worker with pid: 180```''',
-        style=styles.code_container),
+        $ git push plotly master
+        dokku@your-dash-server password:
 
-        rc.Markdown(
+        ```
         '''
-        &nbsp;
-
-        Deployment fails with the above message when you have failed to declare
-        `server` in your `app.py` file. Check your `app.py` file and confirm that
-        you have `server = app.server`.
-
-        &nbsp;
-
-        For more information see
-        <dccLink href="/dash-enterprise/application-structure" children="Application Structure"/>.
+    ),
 
+    rc.Markdown(
+        '''
         &nbsp;
-        ''')
-    ]),
 
-    html.Details([
-        html.Summary("SSH deploy: git push is asking for password."),
-
-        rc.Markdown(
-            '''
-            ```shell
-            $ git push multipage master
-            dokku@dash.local's password:
-            ```
-            ''',
-            style=styles.code_container),
-
-        rc.Markdown(
-            '''
-            &nbsp;
-
-            If you're seeing a request for a password for dokku@your-dash-server, that
-            means that the ssh authentication has failed. This can be for a variety of
-            reasons so it is useful to run git push again with ssh debugging enabled by
-            adding `GIT_SSH_COMMAND='ssh -v'` before your `git push` command.
-            '''),
+        This means that the ssh authentication has failed. 
+        
+        This can be for a variety of
+        reasons so it is useful to run git push again with ssh debugging enabled by
+        adding `GIT_SSH_COMMAND='ssh -v'` before your `git push` command.
```

Comment:
```suggestion
        adding `GIT_SSH_COMMAND='ssh -v'` before your `git push` command:
        ```
        $ GIT_SSH_COMMAND='ssh -v' git push plotly master
        ```
```